Library/Homebrew/extend/os/linux/keg_relocate.rb

```diff
@@ -5,9 +5,6 @@ def relocate_dynamic_linkage(relocation)
     # Patching the dynamic linker of glibc breaks it.
     return if name == "glibc"
 
-    # Patching patchelf using itself fails with "Text file busy" or SIGBUS.
-    return if name == "patchelf"
-
```

Comment:
You can leave this chunk. `patchelf` modifying itself makes me a bit nervous, and it's not needed. The `patchelf` bottle is built not to require relocation.

Reply:
sorry, that was unintentional